### PR TITLE
[quant] Quantizable documentation

### PR DESCRIPTION
**Pull request by HDCharles**
Stack from [ghstack](https://github.com/ezyang/ghstack) (oldest at bottom):
* #71192
* __->__ #71191
* #71190
* #71189

Minor documentation entry for the quantizable LSTM and MHA classes.

Differential Revision: [D33452284](https://our.internmc.facebook.com/intern/diff/D33452284)

### CODE REVIEW

**Review by jerryzh168 — APPROVED** [review]

**Comment by HDCharles** [comment]
@HDCharles has imported this pull request.  If you are a Facebook employee, you can view this diff [on Phabricator](https://www.internalfb.com/diff/D33452284).

**Comment by HDCharles** [comment]
@HDCharles has imported this pull request.  If you are a Facebook employee, you can view this diff [on Phabricator](https://www.internalfb.com/diff/D33452284).

File: docs/source/quantization.rst [hunk 1]
```diff
@@ -39,7 +39,106 @@ APIs are provided that incorporate typical workflows of converting FP32 model
 to lower precision with minimal accuracy loss.
 
 Quantization API Summary
------------------------------
+------------------------
```
Comment (vkuzo) [inline docs/source/quantization.rst:42]: nit: is this new code intended to be in the PR?

**Comment by HDCharles** [comment]
@HDCharles has imported this pull request.  If you are a Facebook employee, you can view this diff [on Phabricator](https://www.internalfb.com/diff/D33452284).

**Comment by HDCharles** [comment]
@HDCharles has imported this pull request.  If you are a Facebook employee, you can view this diff [on Phabricator](https://www.internalfb.com/diff/D33452284).

**Comment by HDCharles** [comment]
@HDCharles has imported this pull request.  If you are a Facebook employee, you can view this diff [on Phabricator](https://www.internalfb.com/diff/D33452284).

**Review by z-a-f — APPROVED** [review]
lg -- one nit: Should we explain what the "quantizable" means, and why it's needed as part of the documentation?

File: docs/source/quantization.rst (hunk 1 above)
Comment (HDCharles) [reply docs/source/quantization.rst]: no

**Comment by HDCharles** [comment]
@HDCharles has imported this pull request.  If you are a Facebook employee, you can view this diff [on Phabricator](https://www.internalfb.com/diff/D33452284).

**Comment by HDCharles** [comment]
@HDCharles has imported this pull request.  If you are a Facebook employee, you can view this diff [on Phabricator](https://www.internalfb.com/diff/D33452284).

**Comment by HDCharles** [comment]
@HDCharles has imported this pull request.  If you are a Facebook employee, you can view this diff [on Phabricator](https://www.internalfb.com/diff/D33452284).

**Pull request closed without merging**